### PR TITLE
Add typings for webostv.js

### PR DESCRIPTION
**Pull request by hcodes**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/webostv/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,1143 @@
+// Type definitions for webostv.js 1.2
+// Project: https://webostv.developer.lge.com/api/webostvjs/intro-webostvjs/
+// Definitions by: Denis Seleznev <https://github.com/hcodes>
+//                 Ilya Istomin <https://github.com/procot>
```
Comment (peterblazejewicz) [inline types/webostv/index.d.ts:4]: you will have to ask Ilya for explicit ack his handle usage here or remove from code owners. Owners here are expected to receive notifications

Comment (hcodes) [reply types/webostv/index.d.ts:4]: /cc @procot Do you agree to be the owner?

**Comment by hcodes** [comment]
Changed the name of the folder.
Before: webostv
After: webostvjs

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by hcodes** [comment]
Ready to merge

File: types/webostv/index.d.ts (hunk 1 above)
Comment (procot) [reply types/webostv/index.d.ts]: Yes of course :)

**Pull request merged**